src/app/lib/config/toggles/localConfig.js

```diff
@@ -26,11 +26,6 @@ export default {
   include: {
     enabled: true,
   },
-  liveRadioSchedule: {
```

Comment:
This can also technically be removed from the test config file, since the toggle exists in the test iSite env.

We might then need another follow up PR to delete from the live config file, but only once we have added the toggle to the live iSite env (if it is even still needed)